packages/body/tests/site/index.html

```diff
@@ -0,0 +1,125 @@
+		</script>
+	<![endif]-->
+</head>
+<body>
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
```

Comment:
Haha you're commenting on the file `packages/body/tests/site/index.html` so yeah... we put it into a test folder. :)
Fair enough, I think it would help me a lot to get you across all the code and how we structured it so far. That way you can add better reviews :)